docs-gen/content/docs/getting-started/installation.md

```diff
@@ -59,6 +59,7 @@ Valid configuration options to override the [Default `values.yaml`]({{< ghblob "
 | `manager.monitoring.serviceMonitor.create`                | Create a ServiceMonitor for the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator)                                                                                                   |
 | `manager.monitoring.serviceMonitor.endpointConfiguration` | Additional configuration to be used in the service monitor endpoint (path, port and scheme are provided)                                                                                                            |
 | `manager.multiversion.enabled`                            | Enable multiversion API. Should be used in production to allow version migration, disable for simplified installation                                                                                               |
+| `manager.multiversion.storedVersion`                      | Must be set if `manager.multiversion.enabled` is set to `true`. It specifies which version of the CRDs should be set as the stored version.                                                                         |
```

Comment:
What about defaulting it to the latest? This way it becomes just an override and it avoidd having to update test values all the time.

Reply:
Done (including docs updates) in 9d8ae851